draft-ietf-quic-multipath.md

```diff
@@ -153,6 +153,9 @@ particular, the multipath extension supports the following scenarios.
     several ones.
   * The client uses multiple IP addresses and the server listens on
     several ones.
+    
+The client can use one or several ports per IP address and the
+server can listen on one ore several ports per IP address. 
```

Comment:
```suggestion
    * The client uses only one IP address and the server
      listens on only one.

Note that in the last scenario, it still remains possible to have
multiple paths over the connection, given that a path is not only
defined by the IP addresses being used, but also the port numbers.
In particular, the client can use one or several ports per IP
address and the server can listen on one or several ports per IP
address.
```

Reply:
As additional context, which I forgot earlier, having multipath from the same IP but different ports is useful for congestion control algorithms that make use of fair queuing: https://github.com/muxamilian/fair-queuing-aware-congestion-control